shell/platform/windows/flutter_windows.cc

```diff
@@ -204,6 +209,11 @@ FlutterDesktopMessengerRef FlutterDesktopRegistrarGetMessenger(
   return registrar->messenger.get();
 }
 
+FlutterDesktopViewRef FlutterDesktopRegistrarGetView(
+    FlutterDesktopPluginRegistrarRef registrar) {
+  return registrar->window;
```

Comment:
Isn't this registrar->GetView() now?

Reply:
`registrar` is a struct, not the class.

Eventually, per previous discussion, I want to eliminate all these structs from the internals in favor of classes, since they are the unfortunate result of a lot of incremental growth in what was supposed to be temporary GLFW code, mirrored over to the Windows implementation.

Reply:
👍